app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -570,12 +570,13 @@ def alert_valid_record?(alert)
         end
       end
     end
-    if %w(mw_heap_used, mw_non_heap_used).include?(@edit.fetch_path(:new, :expression, :eval_method))
+    if %w(mw_heap_used mw_non_heap_used).include?(@edit.fetch_path(:new, :expression, :eval_method))
       value_greater_than = @edit.fetch_path(:new, :expression, :options, :value_mw_greater_than)
       non = @edit.fetch_path(:new, :expression, :eval_method) == "mw_non_heap_used" ? "Non" : ""
       an_integer = "an integer"
       between = "between 0 and 100"
       template_error = "%s %s Heap Max (%%) must be %s"
+      min_max_error = "> %s Heap Max (%%) must be greater than < %s Heap Max (%%)"
```

Comment:
Could you please explain how this error message would look with parameters? 
I'm rereading it a few times and it is unclear to me. (the "heap max but be greater than < X heap max" part confuses me a bit)